### PR TITLE
Enable Sphinx for RTD

### LINKED ISSUE #4: Publish docs to RTD

### PR DESCRIPTION
**Pull request by jamadden**
Also fix usage of zope.contenttype as a main module. (`__init__.py` in a package can never be `__main__`, you need a separate `__main__.py`. If you tried `python -m zope.contenttype`, you'd get "No module named `zope.contenttype.__main__`; 'zope.contenttype' is a package and cannot be directly executed.")

Docs can be viewed at http://zopecontenttype.readthedocs.io/en/docs/

Fixes #4

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+
+
+# -- Options for Texinfo output -------------------------------------------
+
+# Grouping the document tree into Texinfo files. List of tuples
+# (source start file, target name, title, author,
+#  dir menu entry, description, category)
+texinfo_documents = [
+    (master_doc, 'zopecontenttype', u'zope.contenttype Documentation',
+     author, 'zopecontenttype', 'One line description of project.',
```
Comment (icemac) [inline docs/conf.py:330]: `One line description of project`?

**Review by icemac — APPROVED** [review]
Wow! Thank you for even documenting the functions in the code. I like it very much.

File: docs/conf.py (hunk 1 above)
Comment (jamadden) [reply docs/conf.py:330]: That's part of the default template, IIRC. In practice I don't think it matters (who uses the texinfo builder? It's not even an option on RTD.)

**Comment by jamadden** [comment]
Thanks! Can we get a PyPI release (or access) when convenient?

**Comment by icemac** [comment]
@jamadden You are now an owner of `zope.contenttype` on PyPI.

**Comment by jamadden** [comment]
Thank you! 4.3.0 has been pushed to PyPI.

**Pull request merged**